packages/snaps-controllers/src/interface/SnapInterfaceController.ts

```diff
@@ -176,17 +178,21 @@ export class SnapInterfaceController extends BaseController<
    * @param id - The interface id.
    * @param content - The new content.
    */
-  async updateInterface(snapId: SnapId, id: string, content: Component) {
+  async updateInterface(
+    snapId: SnapId,
+    id: string,
+    content: ComponentOrElement,
+  ) {
     this.#validateArgs(snapId, id);
-    await this.#validateContent(content);
+    const element = getJsxInterface(content);
```

Comment:
Any clue how fast this is?

Reply:
Conversion from the old to the new format is very quick, if that's what you mean?